### PR TITLE
Scroll table and top nav fixes

### PR DESCRIPTION
**Pull request by TW80000**
* Fix bug where you can't see last item in table
* Make ScrollTableView row height configurable
  * We need this in Ablii because table rows are 48px tall instead of 40px
* Fix top nav not showing when already logged in (this was a mistake introduced in my old PR that was merged yesterday) 
* Added option to ScrollTableView to make the height dynamic so that it won't go off the bottom of the page

Should fix https://github.com/nanoPayinc/NANOPAY/issues/5158

### CODE REVIEW

File: src/foam/u2/view/ScrollTableView.js [hunk 1]
```diff
@@ -78,20 +107,24 @@
...
+                height: this.rowHeight * limit + this.TABLE_HEAD_HEIGHT,
+                width: 18,
+                size$: this.daoCount$,
+              });
             })).
           end().
         end().
       end();
+
+      if ( this.fitInScreen ) this.onload.sub(this.updateTableHeight);
```
Comment (kgrgreer) [inline src/foam/u2/view/ScrollTableView.js:127]: Should you also listen for resize events? If so, do in a future PR.

File: src/foam/u2/view/ScrollTableView.js [hunk 2]
```diff
@@ -16,6 +16,14 @@
     'foam.u2.view.TableView'
   ],
 
+  constants: [
+    {
+      type: 'Int',
+      name: 'TABLE_HEAD_HEIGHT',
+      value: 40
```
Comment (kgrgreer) [inline src/foam/u2/view/ScrollTableView.js:23]: Could the view figure this out on its own?

File: src/foam/u2/view/ScrollTableView.js (hunk 1 above)
Comment (TW80000) [reply src/foam/u2/view/ScrollTableView.js:127]: Made a ticket for this here: https://github.com/foam-framework/foam2/issues/1736

File: src/foam/u2/view/ScrollTableView.js (hunk 2 above)
Comment (TW80000) [reply src/foam/u2/view/ScrollTableView.js:23]: Included in https://github.com/foam-framework/foam2/issues/1736

**Pull request merged**